src/wp-includes/js/customize-loader.js

```diff
@@ -1,8 +1,5 @@
 /* global _wpCustomizeLoaderSettings, confirm */
-/*
- * Expose a public API that allows the customizer to be
- * loaded on any page.
- */
```

Comment:
Why is this removed? It was added in https://core.trac.wordpress.org/changeset/33709

See https://core.trac.wordpress.org/ticket/41682#comment:14

Reply:
I've replied in the wordpress ticket. This was a subjective call based on feeling the `@namespace` tag conveys the same information. I've no issue with adding similar comments back in.

Reply:
I would prefer all the comments be left in unless they effect the namespace parsing. That way, this patch remains smaller and more focused. We can address these other changes in a follow up ticket if needed.